apps/code-examples/src/app/code-examples/forms/input-box/basic/input-box-demo.component.html

```diff
@@ -58,8 +58,7 @@ <h2>New member form</h2>
             <label class="sky-control-label" [for]="bioInput.id"> Bio </label>
             <sky-help-inline
               class="sky-control-help"
-              [skyPopover]="bioHelpPopover"
-              skyPopoverTrigger="click"
```

Comment:
this code example does work but we should make another code example so that inline help and the basic input arent in the same code example